migtests/lib/yb.py

```diff
@@ -24,6 +29,14 @@ def new_target_db():
 		env.get("TARGET_DB_PASSWORD", ""),
 		env["TARGET_DB_NAME"])
 
+def new_source_db():
```

Comment:
This is for source_replica? so function name and db_type should be different I suppose?